docs/core/whats-new/index.md

```diff
@@ -0,0 +1,155 @@
+
+**Expanded surface area**
+
+The total number of APIs available on .NET Core 2.0 has more than doubled in comparison with .NET Core 1.1. 
+
+**Support for .NET Framework libraries**
+
+.NET Core code can reference existing .NET Framework libraries, including existing NuGet packages. Note that the libraries must use APIs that are found in .NET Standard.
+
+## Visual Studio integration
+
+Visual Studio 2017 version 15.3 and in some cases Visual Studio for Mac offer a number of significant enhancements for .NET Core developers.
+
+**Retargeting .NET Core apps and .NET Standard libraires**
+
+If the .NET Core 2.0 SDK is installed, projects that target .NET Core 1.x can be retargeted to .NET Core 2.0, and libraires that target .NET Standard 1.6 or earlier can be retargeted to .NET Standard 2.0. To retarget your project, you open the **Application** tab of the project's properties dialog and change the **Target framework** value to **.NET Core 2.0** or **.NET Standard 2.0**. You can also change it by right-clicking on the project and selecting the **Edit *.csproj file** option. For more information, see the [Tooling](#tooling) section earlier in this topic.
+
+**Live Unit Testing support for .NET Core**
+
+Whenever you modify your code, Live Unit Testing automatically runs any affected unit tests in the background and displays the results and code coverage live in the Visual Studio environment. .NET Core 2.0 now supports Live Unit Testing. Previously, Live Unit Testing was available only for .NET Framework applications. 
+
+For more information, see [Live Unit Testing with Visual Studio 2017](https://docs.microsoft.com/en-us/visualstudio/test/live-unit-testing) and the [Live Unit Testing FAQ](https://docs.microsoft.com/en-us/visualstudio/test/live-unit-testing-faq).
+
+**Better support for multiple target frameworks**
+
+If you're building a project for multiple target frameworks, you can now select the target platform from the top-level menu. In the following figure, a project named SCD1 targets 64-bit Mac OS X 10.11 (`osx.10.11-x64`) and 64-bit Windows 10/Windows Server 2016 (`win10-x64`). You can select the target framework before selecting the project button, in this case to run a debug build.
+
+![Selecting the target framework when building a project](media/multitarget.png) 
+
+**Side-by-side support for .NET Core SDKs**
+
+You can now install the .NET Core SDK independently of Visual Studio. This makes it possible for a single version of Visual Studio to build projects that target different versions of .NET Core. Previously, Visual Studio and the .NET Core SDK were tightly coupled; a particular version of the SDK accompanied a particular version of Visual Studio.
+
+## Documentation improvements
+
+**.NET Application Architecture**
```

Comment:
are we adding this here for visibility? is the guide created specifically for 2.0?